packages/@react-aria/listbox/src/useOption.ts

```diff
@@ -24,7 +24,10 @@ interface OptionAria {
   labelProps: HTMLAttributes<HTMLElement>,
 
   /** Props for the description text element inside the option, if any. */
-  descriptionProps: HTMLAttributes<HTMLElement>
+  descriptionProps: HTMLAttributes<HTMLElement>,
+
+  /** Props for the hover style. */
+  isHovered: boolean
```

Comment:
Usually we try to keep the return values for component hooks limited to DOM props. Since the hover styling here is specific to Spectrum (not all designs will necessarily have one), I think it makes sense for the `useHover` call to be in the `ListBoxOption` component instead of in the `useOption` hook. I know there's already a `useHover` call here, but since the `isHovered` boolean is being used for styling purposes, I think it makes sense to put it in the component. Should be ok to have two of them.

Same comment for the other hooks below that you added `isHovered` to.

Reply:
ah, I like that option

Reply:
Nice! To think of hooks' reusability, 
it was a bad approach to add `isHovered` to them directly. Thank you.

Reply:
@devongovett 
The problem I found is both ListBox and Picker uses useHover for different behavior, and they conflict with each other.
https://github.com/adobe/react-spectrum/pull/775#discussion_r456831098